src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/mediapicker/mediapicker.controller.js

```diff
@@ -29,7 +29,6 @@ angular.module("umbraco")
             var dialogOptions = $scope.model;
 
             $scope.disableFolderSelect = (dialogOptions.disableFolderSelect && dialogOptions.disableFolderSelect !== "0") ? true : false;
-            $scope.disableFocalPoint = (dialogOptions.disableFocalPoint && dialogOptions.disableFocalPoint !== "0") ? true : false;
```

Comment:
@lars-erik note that this is passed in from here and it would be a breaking change to remove as devlopers might use the infinite `mediapicker`.
https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Web.UI.Client/src/common/services/tinymce.service.js#L1564

Reply:
I'm aware, but it's not used for anything beyond this point. If you want the cropper, you provide a cropSize, otherwise you only get a preview. I don't think the focal point was ever saved. I'm hoping to look into whether it's possible to allow cropping from the RTE as well. In any case I think this field is obsolete and will remove it completely if we do the above.